x-pack/packages/kbn-langchain/server/language_models/chat_openai.ts

```diff
@@ -147,7 +147,13 @@ export class ActionsClientChatOpenAI extends ChatOpenAI {
       const actionResult = await this.#actionsClient.execute(requestBody);
 
       if (actionResult.status === 'error') {
-        throw new Error(`${LLM_TYPE}: ${actionResult?.message} - ${actionResult?.serviceMessage}`);
+        const error = new Error(
+          `${LLM_TYPE}: ${actionResult?.message} - ${actionResult?.serviceMessage}`
+        );
+        if (actionResult?.serviceMessage) {
+          error.name = actionResult?.serviceMessage;
```

Comment:
`generateChatTitle` will first look for `error.name` and if not fall back to `error.message`, in order to deliver a terse title. Therefore I'm putting the `serviceMessage` (comes from LLM directly) on `error.name` in each of these chat models.